### ISSUE TITLE
Stream to usb directly

### ISSUE THREAD

**Issue by kekkokk** [webosbrew/hyperion-webos#114]
Hi everyone,
I’ve seen there’s an interesting project called serialwled and since I have lots of interference in my network I wanted to know if there’s a way that I can plug the esp32 directly to the tv usb instead of using the network.
Thanks!

**Comment by satgit62** [comment]
Yes, you can do this with HyperSerial from AWAWA see: https://github.com/awawa-dev/HyperSerialESP32.
Depending on which ESP you use, you may need to install the necessary driver on your LG. Drivers for each TV architecture can be found on this page: https://github.com/throwaway96/webos-kernel-drivers

**Comment by Fietspomp86** [comment]
I'm using a Lightberry, which is basically an Arduino Leonardo, directly connected to the TV USB port.
Works like a charm with 218 APA102 LEDs.

Lightberry is flashed with code from the Hyperion github, using Fastled.

No any extra drivers needed.
